src/pages/UpdateServers.js

```diff
@@ -93,6 +102,64 @@ class UpdateServers extends BaseUpdateWizardPage {
           };
         });
       });
+    this.checkMonasca();
+  }
+
+  /**
+   * checks to see if Monasca is installed, and if it is, triggers a call to the status
+   * of each server in the model
+   */
+  checkMonasca() {
+    if(this.state.monasca === undefined) {
+      fetchJson('/api/v1/clm/monasca/is_installed')
+        .then(responseData => {
+          if(responseData.installed) {
+            getInternalModel()
+              .then((yml) => {
+                this.setState({internalModel: yml, monasca: true}, () => this.getServerStatuses());
+              }).catch((error) => {
+                console.log('error getting internal model:' + error);// eslint-disable-line no-console
+                this.setState({monasca: true}); //if the model doesnt load but monasca is present,
+                //update the state accordingly, but do not try to load server statuses from an undefined model
+              });
+          } else {
+            this.setState({monasca: false});
+          }
+        }).catch((error) => {
+          console.log('error checking if Monasca is installed:' + error);// eslint-disable-line no-console
+        });
+    } else if(this.state.monasca === true) {
+      //if monasca is installed, get the server statuses
+      this.getServerStatuses();
+    }
+  }
+
+  /**
+   * get the monasca status (up/down/unknown) for each server in the cloud
+   */
+  getServerStatuses() {
+    if(this.state.monasca && this.state.internalModel !== undefined
+        && this.props.model !== undefined) {
+      // get the list of all servers, then load their statuses
+      // possible future enhancement: batching this
+      const serverIds = this.props.model.getIn(['inputModel','servers'])
+        .map(server => server.get('id'));
+
+      serverIds.forEach(async (server_id) => {
+        let server = this.state.internalModel.internal.servers.find(s => s.id == server_id);
+        await fetchJson('/api/v1/clm/monasca/server_status/' + server.hostname)
```

Comment:
using await here isn't really beneficial at all. I think what you want is:
```js
let responceData = await fetchJson(`/api/v1/clm/monasca/server_status/${server.hostname}`);
```
Then you can get rid of the `then` and catch calls